testsuite/rust_generator/tests/AC/test.yml

```diff
@@ -1,4 +1,5 @@
 language: PY3
+executors: [RUST]
```

Comment:
Can we rename this to `required_auxiliary_executors` so it's clear what this is for?

Reply:
Renamed.